### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by ronbak**
Hi,

this merge request fixes some minor issues.
It was created with help of [Feram][repo].

If it helped to improve this repository,
please consider to [star Feram's repo][repo] to show some love! 😊

PS:
The [Feram team][feramhq] is currently working on making this technology
available for everyone. If you want to be one of the first to use it,
please sign up up for the newsletter on [feram.co].

[repo]: https://github.com/feramhq/feram
[feramhq]: https://github.com/feramhq
[feram.co]: http://feram.co

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,5 +43,17 @@
   },
   "engines": {
     "node": ">= 0.10"
+  },
+  "keywords": [
+    "hypernova",
+    "client"
+  ],
+  "directories": {
```
Comment (ljharb) [inline package.json]: what is this package.json key for? i'm not aware of it being a typical npm key

Comment (ljharb) [inline package.json]: i found https://docs.npmjs.com/files/package.json#directories , but this field isn't used for anything, so i'd rather omit it.

Comment (ronbak) [inline package.json]: Well, some tools reference it … but yeah, it's not really widespread yet. Kind of a catch-22 …

**Comment by ljharb** [comment]
Since this PR doesn't allow maintainers to edit; there's been no response; and unsolicited PRs created by automated tools are not very polite; I'm going to close this.

**Pull request closed without merging**